linux/devdoc/threadpool_linux_requirements.md

```diff
@@ -129,6 +129,15 @@ MOCKABLE_FUNCTION(, THANDLE(THREADPOOL), threadpool_create, EXECUTION_ENGINE_HAN
 
 **SRS_THREADPOOL_LINUX_07_010: [** `insert_idx` and `consume_idx` for the task array shall be initialized to 0. **]**
 
+[Will be moved from threadpool_open]
+**S_R_S_THREADPOOL_LINUX_11_001: [** `threadpool_create` shall initialize internal threapool data items **]**
```

Comment:
This is very vague, I would have no idea how to write a test for it.

Reply:
I think it should be removed

Reply:
Okay, I think there was already some test for it :)

Reply:
Confirmed there is a good test for this scenario.

Reply:
I assume you are talking about the original
SRS_THREADPOOL_LINUX_11_001. I see that is tagged in the Codes for initializing with 0 and respectively setting the state for each entry in the array that is owned by the threadpool.

I understand you copied it from open, but that is still wrong. If Jelani would send this for review today I would make the same comment.

The point here is that just saying initialize internal threadpool data items would not describe at all what we would want to test.
We might want to test that we set the state for each element to NOT_USED - that is if we truly describe that there is a state associated with each element in the array.

Please ping so we discuss if you did not understand what I mean here -  we should clarify these so we don't leave holes in knowledge behind.